pysilcam/tests/test_process.py

```diff
@@ -101,4 +162,4 @@ def test_output_files():
     #os.makedirs(reportdir, exist_ok=True)
     #synth.generate_report(os.path.join(reportdir, 'imagesynth_report.pdf'), PIX_SIZE=28.758169934640524,
     #                      PATH_LENGTH=10, d50=800, TotalVolumeConcentration=800,
-    #                      MinD=108)
+    #                      MinD=108)
```

Comment:
Note that it is good practice to have a newline at the end of the file. Some tools rely on it. Therefore linters tend to warn this, and so does GitHub indicate (marked with 🚫 ).